src/configs/index.js

```diff
@@ -426,3 +426,5 @@ export const NOTHING_FOUND_MESSAGE = ['Нічого не знайдено', 'Not
 export const commentsLimit = 10;
 
 export const DEFAULT_SIZE = 'M';
+
+export const ERROR_PAGE = '/error-page';
```

Comment:
Please move this to` routes.js` file
![image](https://user-images.githubusercontent.com/59802190/106784749-5a68c100-6655-11eb-8476-ff97fb4816a1.png)

Reply:
fixed